test/boost/tablets_test.cc

```diff
@@ -2452,6 +2452,7 @@ SEASTAR_THREAD_TEST_CASE(test_load_balancing_with_random_load) {
                     // Leave the first host empty by making it invisible to allocation algorithm.
                     hosts_by_rack[rack.rack].push_back(h);
                 }
+                co_await tm.update_normal_tokens(std::unordered_set{token(tests::d2t(double(i) / hosts.size()))}, h);
```

Comment:
It looks like it belongs to a different change.

Reply:
Right.